Automattic/wp-calypso
# Hide buttons and disable links to switch theme from P2020

#### Changes proposed in this Pull Request
Switching the theme from a P2020 site causes some problems. This PR disables that possibility from Calypso.

**Before (on a P2020 site):**
<img width="1263" alt="bef" src="https://user-images.githubusercontent.com/193283/145799508-1417c00b-a5f1-4e4b-b97b-a3c77e645e3c.png">

**After (on a P2020 site):**
<img width="1251" alt="aft" src="https://user-images.githubusercontent.com/193283/145799556-8b63a931-1470-4c2d-ad5d-f7f52b67f2e0.png">
#### Testing instructions

Use the Seedlet theme as the theme to switch to for instance: Go to http://calypso.localhost:3000/theme/seedlet/[some p2020 site]
* You should not see the "Activate this design" or the "Open live demo" button.
* Clicking the preview to the right should do nothing. It should not open the preview iframe.

On a non-p2020 site:
* Check that you get the buttons and that the preview opens an iframe when clicked.


See 1127-gh-Automattic/p2
